shared/message/motion/KickCommand.proto

```diff
@@ -42,8 +42,8 @@ message KickCommand {
  * @author Brendan Annable
  */
 message KickScriptCommand {
-    vec3   direction = 1;  // Direction to kick with magnitude determining force
-    uint32 leg       = 2;  // Leg to kick with
+    uint32 leg       = 1;  // Leg to kick with
+    KickCommandType type = 2;
```

Comment:
Type is 2 but only 0 and 1 have been defined in the KickCommandType enum?

Reply:
When you define protobuf messages, you're basically giving the (sub)message or field type, and the order they appear in this message. 
So, the `= 1;`, and `= 2;` in this `.proto` file are defining the order that the fields are in the message, not the actual values they'll take. Those values are selected when we're creating the message to send it. E.g. [here](https://github.com/NUbots/NUbots/blob/master/module/behaviour/planning/KickPlanner/src/KickPlanner.cpp#L184) is an example using this message. In the message constructor, the order of the arguments is the same as defined here. You can also set them manually like [here](https://github.com/NUbots/NUbots/blob/master/module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.cpp#L275)